src/_tutorials/web/images/portmanteaux-code.svg

```diff
@@ -0,0 +1,3 @@
29wZW5jbGlwYXJ0Lm9yZy9kZXRhaWwvMTY2MzU2L21vdXNlLWN1cnNvci0tLWFycm93LWJ5LWhlbGxvY2F0Zm9vZDwvZGM6c291cmNlPiYjeGE7ICAgICAgICA8ZGM6Y3JlYXRvcj4mI3hhOyAgICAgICAgICA8Y2M6YWdlbnQ+JiN4YTsgICAgICAgICAgICA8ZGM6dGl0bGU+aGVsbG9jYXRmb29kPC9kYzp0aXRsZT4mI3hhOyAgICAgICAgICA8L2NjOmFnZW50PiYjeGE7ICAgICAgICA8L2RjOmNyZWF0b3I+JiN4YTsgICAgICAgIDxkYzpzdWJqZWN0PiYjeGE7ICAgICAgICAgIDxyZGY6YmFnPiYjeGE7ICAgICAgICAgICAgPHJkZjpsaT5hcnJvdzwvcmRmOmxpPiYjeGE7ICAgICAgICAgICAgPHJkZjpsaT5jdXJzb3I8L3JkZjpsaT4mI3hhOyAgICAgICAgICAgIDxyZGY6bGk+bW91c2U8L3JkZjpsaT4mI3hhOyAgICAgICAgICAgIDxyZGY6bGk+cGl4ZWw8L3JkZjpsaT4mI3hhOyAgICAgICAgICA8L3JkZjpiYWc+JiN4YTsgICAgICAgIDwvZGM6c3ViamVjdD4mI3hhOyAgICAgIDwvY2M6d29yaz4mI3hhOyAgICAgIDxjYzpsaWNlbnNlIHJkZjphYm91dD0iaHR0cDovL2NyZWF0aXZlY29tbW9ucy5vcmcvcHVibGljZG9tYWluL3plcm8vMS4wLyI+JiN4YTsgICAgICAgIDxjYzpwZXJtaXRzIHJkZjpyZXNvdXJjZT0iaHR0cDovL2NyZWF0aXZlY29tbW9ucy5vcmcvbnMjUmVwcm9kdWN0aW9uIi8+JiN4YTsgICAgICAgIDxjYzpwZXJtaXRzIHJkZjpyZXNvdXJjZT0iaHR0cDovL2NyZWF0aXZlY29tbW9ucy5vcmcvbnMjRGlzdHJpYnV0aW9uIi8+JiN4YTsgICAgICAgIDxjYzpwZXJtaXRzIHJkZjpyZXNvdXJjZT0iaHR0cDovL2NyZWF0aXZlY29tbW9ucy5vcmcvbnMjRGVyaXZhdGl2ZVdvcmtzIi8+JiN4YTsgICAgICA8L2NjOmxpY2Vuc2U+JiN4YTsgICAgPC9yZGY6cmRmPiYjeGE7ICA8L21ldGFkYXRhPiYjeGE7PC9zdmc+" preserveAspectRatio="none" pointer-events="none"/><rect x="350" y="120" width="140" height="20" fill="#000000" stroke="#000000" pointer-events="none" transform="translate(2,3)" opacity="0.25"/><rect x="350" y="120" width="140" height="20" fill="#f5f5f5" stroke="#666666" pointer-events="none"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 138px; height: 1px; padding-top: 130px; margin-left: 351px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: #333333; line-height: 1.2; pointer-events: none; white-space: normal; word-wrap: normal; "><font style="font-size: 15px">mouse-click handler</font></div></div></div></foreignObject><text x="420" y="134" fill="#333333" font-family="Helvetica" font-size="14px" text-anchor="middle">mouse-click handler</text></switch></g><path d="M 343.66 129.43 L 73.35 105" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 348.89 129.9 L 341.6 132.76 L 343.66 129.43 L 342.23 125.78 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
Could you perhaps make the code box a little larger? Seems like the beginning and end are cutting off causing some artifacts.

![image](https://user-images.githubusercontent.com/18372958/121728484-f054a280-cab2-11eb-83f4-10313add981f.png)